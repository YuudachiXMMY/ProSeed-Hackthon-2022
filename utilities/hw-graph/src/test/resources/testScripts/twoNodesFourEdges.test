# Test to add two nodes to graph and add edges that is between them and points to themselves

# Create a graph
CreateGraph graph2

# Add two nodes to graph
AddNode graph2 node1
AddNode graph2 node2

# Add edges to graph
AddEdge graph2 node1 node1 edge1
AddEdge graph2 node1 node2 edge2
AddEdge graph2 node2 node2 edge3
AddEdge graph2 node2 node1 edge4

# Print all nodes in graph
ListNodes graph2
ListChildren graph2 node1
ListChildren graph2 node2