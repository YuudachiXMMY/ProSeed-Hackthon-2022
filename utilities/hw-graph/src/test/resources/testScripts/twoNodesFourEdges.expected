# Test to add two nodes to graph and add edges that is between them and points to themselves

# Create a graph
created graph graph2

# Add two nodes to graph
added node node1 to graph2
added node node2 to graph2

# Add edges to graph
added edge edge1 from node1 to node1 in graph2
added edge edge2 from node1 to node2 in graph2
added edge edge3 from node2 to node2 in graph2
added edge edge4 from node2 to node1 in graph2

# Print all nodes in graph
graph2 contains: node1 node2
the children of node1 in graph2 are: node1(edge1) node2(edge2)
the children of node2 in graph2 are: node1(edge4) node2(edge3)